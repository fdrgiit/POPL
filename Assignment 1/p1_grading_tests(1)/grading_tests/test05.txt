
00123INT1234.9002REAL

