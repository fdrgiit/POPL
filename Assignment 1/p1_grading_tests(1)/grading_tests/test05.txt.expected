2 34 0 
2 34 0 
2 34 123 
2 36 INT1234.9002REAL 
